### ISSUE TITLE
View My files too many redirects

### ISSUE THREAD

**Issue by GoogleCodeExporter** [srulad/clients-oriented-ftp#244]
```
What steps will reproduce the problem?
1.Clicking on View My Files produces error
2.
3.

What is the expected output? What do you see instead?

Error 310 (net::ERR_TOO_MANY_REDIRECTS): There were too many redirects.

What version of the product are you using? On what operating system?

 version r375 

Please provide any additional information below.

```

Original issue reported on code.google.com by `sro...@gmail.com` on 25 Mar 2013 at 7:58

**Comment by GoogleCodeExporter** [comment]
```
Upon user login. Redirect problem occurs. Chrome, Safari, Firefox on MAC 10.7.5 
going to example.com/upload-from-computer.php allow me to access the user panel.
```

Original comment by `sro...@gmail.com` on 25 Mar 2013 at 10:54

**Comment by GoogleCodeExporter** [comment]
```
I am having the same issue with version r412. When I view the demo pages the 
issue does not occur but when I set it up on a local server the pages took 
forever to load in explorer (since it doesn't stop after so many redirects) and 
stopped after around 20+ endings of /my_files in the url. I tried to look 
through the code to find the bug but haven't fount it yet. If anyone does 
please let us know so I can fix it!
```

Original comment by `Dubz31...@gmail.com` on 1 May 2013 at 7:22

**Comment by GoogleCodeExporter** [comment]
```
I got this only once using r375
It was local using wampserver / Chrome
I was switching from 2 projectsend installations during my tests 
so I think this is why I got this error message, I'm not able to reproduce this 
bug.
```

Original comment by `access_c...@hotmail.com` on 2 May 2013 at 4:42

**Comment by GoogleCodeExporter** [comment]
```
^ That could be the problem.

The redirect is probably being cause by cookies. If you clear all cookies and 
then login without selecting "Remember me", does it work?
```

Original comment by `i...@subwaydesign.com.ar` on 2 May 2013 at 4:48

**Comment by GoogleCodeExporter** [comment]
```
I realized after a while that the .htaccess file on the root of the site was 
taking over the url. I had it set to take the end of the url (ex: 
localhost/test would take test) and place it in a get variable so it would 
reroute it to something like [index.php?page=$1]. After noticing that linux 
(ubuntu specifically) hodes dot files (such as .htaccess) I removed the file 
and it's working seamlessly now.
```

Original comment by `Dubz31...@gmail.com` on 2 May 2013 at 7:59

**Comment by GoogleCodeExporter** [comment]
```
I had r412 running next to a wordpress site, with the same error. The .htaccess 
file of the root directory had some rewrite rules, figured it had something to 
do with that. All I did was write a .htaccess file in the /projectsend/ 
directory with the following contents:

<IfModule mod_rewrite.c>
    RewriteEngine Off
</IfModule>

Solved it for me
```

Original comment by `vuokko.v...@gmail.com` on 5 Jun 2013 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
GREAT SOLUTION!!!
I'm including this in the next version!!
```

Original comment by `i...@subwaydesign.com.ar` on 5 Jun 2013 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
The rewriting didnt fully solve it, there's another issue with this: if a user 
has a login 'Foo' but enter 'foo' the database query will return a user, and 
store 'foo' in the sessions, but i guess this causes the redirects because when 
I change the query in 'index.php' to read like this:

        /** Look up the system users table to see if the entered username exists */
        $sql_user = $database->query("SELECT * FROM tbl_users WHERE BINARY user='$sysuser_username'");

(case sensitive query) the issue is resolved 
```

Original comment by `vuokko.v...@gmail.com` on 5 Jun 2013 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
Having the same issue, any luck in solving it?
```

Original comment by `concierg...@gmail.com` on 10 Jun 2013 at 4:38

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Watch the new Github repo at https://github.com/ignacionelson/ProjectSend , 
have submitted a possible fix via PR

/necro
```

Original comment by `pureri...@gmail.com` on 26 Mar 2015 at 9:40
